liwords-ui/package.json

```diff
@@ -18,6 +18,7 @@
     "@types/react-dom": "^18.3.5",
     "antd": "^5.22.4",
     "browserslist-to-esbuild": "^2.1.1",
+    "dayjs": "^1.11.13",
```

Comment:
this was already depended on by antd so it shouldn't increase the bundle or anything, just making the dependency explicit now that we actually use the library ourselves